recipe/test_script_exist.bat

```diff
@@ -0,0 +1,25 @@
+REM As a workaround for GDAL apps unexpectedly returning error codes, this
+REM script checks only for the existence of the .py files below.
+
+@echo on
+
+echo Test existence of gdalattachpct.py...
+if not exist %LIBRARY_BIN%\\gdalattachpct.py exit 1
```

Comment:
Single back slash should do it. You just need double in the yaml.

Reply:
True, though it should work in practice in most cases.